plugins/org.python.pydev.debug/src_console/org/python/pydev/debug/newconsole/env/ChooseProcessTypeDialog.java

```diff
@@ -242,6 +242,7 @@ protected void okPressed() {
             natures.add(nature);
             IInterpreterManager relatedInterpreterManager = nature.getRelatedInterpreterManager();
             this.interpreterManager = relatedInterpreterManager;
+            InteractiveConsolePrefs.setPydevAsInterpreter();
```

Comment:
It shouldn't always do this. Please create a checkbox with a `Save selection as default?` and if it's selected, on `okPressed`, save the setting based on what was selected.